packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -81,7 +81,7 @@ import {
 } from "./mergeTreeNodeWalk";
 import type { TrackingGroup } from "./mergeTreeTracking";
 import { zamboniSegments } from "./zamboni";
-import { Client, reservedMarkerIdKey } from ".";
+import { Client, reservedMarkerIdKey, IRemovalInfo } from ".";
```

Comment:
can you add the new import from the file directly, relative imports can get nasty over time. Also @Abe27342 i'm a bit worried about the cirucular ref here with Client. seems needed for attribution. just something to clean up.

Reply:
ah.. yeah should be possible to clean that up by lifting things to client.